en/mkdocs.yml

```diff
@@ -1826,7 +1827,7 @@ plugins:
               'learn/design-api/create-api/generate-rest-api-from-soap-backend.md': 'https://apim.docs.wso2.com/en/4.1.0/design/create-api/create-rest-api/generate-rest-api-from-soap-backend/'
               'learn/design-api/create-api/test-a-rest-api.md': 'https://apim.docs.wso2.com/en/4.1.0/design/create-api/create-rest-api/test-a-rest-api/'
               'learn/api-gateway/overview-of-the-api-gateway.md': 'https://apim.docs.wso2.com/en/4.1.0/deploy-and-publish/deploy-on-gateway/api-gateway/overview-of-the-api-gateway/'
-              'learn/api-gateway/message-mediation/changing-the-default-mediation-flow-of-api-requests.md': 'https://apim.docs.wso2.com/en/4.1.0/deploy-and-publish/deploy-on-gateway/api-gateway/message-mediation/changing-the-default-mediation-flow-of-api-requests/'
+            #   'learn/api-gateway/message-mediation/changing-the-default-mediation-flow-of-api-requests.md': 'https://apim.docs.wso2.com/en/4.1.0/deploy-and-publish/deploy-on-gateway/api-gateway/message-mediation/changing-the-default-mediation-flow-of-api-requests/'
```

Comment:
Please add redirects for all pages so that all the previous links are not broken. There are one or two that are broken even in the current docs as we haven't added those in when they were under the Deploy and Publish section. There is an open issue.

Reply:
Fixed the commented out entry with https://github.com/wso2/docs-apim/pull/5846/commits/1a085cb0cae7756f4da82ad6d1f14f35758b6d6e. Will address the other broken links with a separate PR.